tools/tertiary-analysis/data-scxa/retrieve-scxa.xml

```diff
@@ -5,13 +5,22 @@
     </macros>
     <expand macro="requirements" />
     <command detect_errors="exit_code"><![CDATA[
-        get_experiment_data.R --accesssion-code "${accession_code}" --matrix-type "${matrix_type}" --get-sdrf "${get_sdrf}" --get-condensed-sdrf "${get_condensed_sdrf}" --get-marker-genes "${get_marker_genes}" 
+        ln -s "${accession_code}_${matrix_type}/10x_data/matrix.mtx" matrix.mtx &&
```

Comment:
In these cases I tend to move directly into the output variable (and that spares you the extra from_work_dir), as the working directory will be deleted at the end of the job, but if this is working it means that Galaxy is grabbing it before the cleanup, so it should be fine.

Reply:
wouldn't it be more natural to do soft links after running the command? at this point the files don't exist yet, it didn't complain because of that?

Reply:
@pcm32 Symlinks work fine with non-existent files, I've checked it. The reason why I put them before the main script is that we don't know in advance which if-block will be the last one, so we can't correctly place `&&` in advance to combine two commands.